indexers/keyvalue/RedisDBIndexer/README.md

```diff
@@ -1,3 +1,45 @@
 # RedisDBIndexer
 
-Use Redis as a key-value indexer 
\ No newline at end of file
+Key-value indexer wrapper around the Redis database.  
+
+## Snippets:
+
+Initialise RedisDBIndexer:
+
+`RedisDBIndexer(hostname=host_ip, port=host_port, db=db_int)`
+
+Users can use Pod images in several ways:
+
+- Run with Docker (`docker run`)
+  
+  ```bash
+    docker run jinahub/pod.indexer.redisdbindexer:MODULE_VERSION-JINA_VERSION --port-in 55555 --port-out 55556
+    ```
+
+- Flow API
+  
+  ```python
+    from jina.flow import Flow
+    f = (Flow()
+        .add(name='my-indexer', image='jinahub/pod.indexer.redisdbindexer:MODULE_VERSION-JINA_VERSION', port_in=55555, port_out=55556)
```

Comment:
I think we are not using `image` anymore and we are using `uses=‘docker://` instead

Reply:
ah yes thanks